### ISSUE TITLE
Does this transform packages?

### ISSUE THREAD

**Issue by jrote1** [dart-lang/async_await#82]
I have a package that uses async and is used in another project but dosent seem to be converting the async stuff for the package. Is this by design?

**Comment by floitschG** [comment]
I'm not sure how this could be easily handled with transformers, but since dart2js will soon have native support I doubt it will be fixed in this package.

**Comment by munificent** [comment]
> I have a package that uses async and is used in another project but dosent seem to be converting the async stuff for the package. Is this by design?

No, this should just work. Can you point me to your package, or at least its pubspec?
